integration_tests/dbt_project.yml

```diff
@@ -81,16 +81,16 @@ vars:
 
   pinterest_source:
     pinterest_schema: ad_reporting_integration_tests_5
-    pinterest_ad_group_history_identifier: "pinterest_ads_ad_group_history_data"
-    pinterest_campaign_history_identifier: "pinterest_ads_campaign_history_data"
-    pinterest_pin_promotion_history_identifier: "pinterest_ads_pin_promotion_history_data"
-    pinterest_pin_promotion_report_identifier: "pinterest_ads_pin_promotion_report_data"
-    pinterest_keyword_history_identifier: "pinterest_ads_keyword_history_data"
-    pinterest_keyword_report_identifier: "pinterest_ads_keyword_report_data"
-    pinterest_ad_group_report_identifier: "pinterest_ads_ad_group_report_data"
-    pinterest_campaign_report_identifier: "pinterest_ads_campaign_report_data"
-    pinterest_advertiser_history_identifier: "pinterest_ads_advertiser_history_data"
-    pinterest_advertiser_report_identifier: "pinterest_ads_advertiser_report_data"
+    pinterest_ad_group_history_identifier: "pinterest_ad_group_history_data"
+    pinterest_campaign_history_identifier: "pinterest_campaign_history_data"
+    pinterest_pin_promotion_history_identifier: "pinterest_pin_promotion_history_data"
+    pinterest_pin_promotion_report_identifier: "pinterest_pin_promotion_report_data"
+    pinterest_keyword_history_identifier: "pinterest_keyword_history_data"
+    pinterest_keyword_report_identifier: "pinterest_keyword_report_data"
+    pinterest_ad_group_report_identifier: "pinterest_ad_group_report_data"
+    pinterest_campaign_report_identifier: "pinterest_campaign_report_data"
+    pinterest_advertiser_history_identifier: "pinterest_advertiser_history_data"
+    pinterest_advertiser_report_identifier: "pinterest_advertiser_report_data"
```

Comment:
Is there a reason these were all renamed? Is it just to be more in line with the actual name of the package? If so, I am on board with that, but remember that the [seed configs for Pinterest](https://github.com/fivetran/dbt_ad_reporting/blob/71a3626968b7a4e261984111457a1b3f905f8964/integration_tests/dbt_project.yml#L286-L375) will also need to be updated to reflect these changes.

Reply:
Yep just to make it standard... thanks for reminding to change it in the configs too!